src/google/protobuf/compiler/csharp/csharp_options.h

```diff
@@ -68,9 +69,12 @@ struct Options {
   // Whether the generated classes should have accessibility level of "internal".
   // Defaults to false that generates "public" classes.
   bool internal_access;
-  // Whether the generated classes should have a global::System.Serializable attribute added
-  // Defaults to false
+  // Whether the generated classes should have a global::System.Serializable attribute added.
+  // Defaults to false.
   bool serializable;
+  // Whether the generated files are annotated with "#nullable enable" to display warnings on potential null assignments. 
```

Comment:
The comment still only talks about `#nullable enable`, which is not true anymore.